src/gmt_remote.c

```diff
@@ -274,7 +274,7 @@ GMT_LOCAL int gmthash_get_url (struct GMT_CTRL *GMT, char *url, char *file, char
 	return 0;
 }
 
-struct GMT_DATA_HASH * hash_load (struct GMT_CTRL *GMT, char *file, int *n) {
+GMT_LOCAL struct GMT_DATA_HASH * hash_load (struct GMT_CTRL *GMT, char *file, int *n) {
```

Comment:
Aren't all GMT_xx public?

Reply:
See #2996 for this discussion - you are both right and wrong on these comments; I will resolve them for now.